### PR TITLE
Language of the D-pressed

### PR DESCRIPTION
**Pull request by Siianko**
### PT Story: 
https://www.pivotaltracker.com/story/show/159733230
```
As a user, 
In order to better understand the content on the web, 
I would like to have some language options. 
```

### CODE REVIEW

File: config/locales/se.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  article_successfully_created: Artikel skapad.
+  published_on: Publicerad 
+  Registration: Registrering
+  password: Lösenord
+  characters_minimum: minimum tecken
+  new-password: Nytt lösenord
+  remember_me: Kommer ihåg mig
+  Sign_up: Signa upp mig
+  password_confirmation: lösenordsbekräftelse
+  Log_in: Logga in
```
Comment (Andycutter) [inline config/locales/se.yml]: log_in instead of Log_in ;-)

File: config/locales/se.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+se: 
+  flash[:error]: Fält kan inte vara tomma. Din artikel kunde inte sparas
+  article_successfully_created: Artikel skapad.
+  published_on: Publicerad 
+  Registration: Registrering
+  password: Lösenord
+  characters_minimum: minimum tecken
+  new-password: Nytt lösenord
+  remember_me: Kommer ihåg mig
+  Sign_up: Signa upp mig
```
Comment (Andycutter) [inline config/locales/se.yml]: sign_up so everything is with small letters.
We need to change Sign_up to sign_up in the views though

File: config/locales/se.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+se: 
+  flash[:error]: Fält kan inte vara tomma. Din artikel kunde inte sparas
+  article_successfully_created: Artikel skapad.
+  published_on: Publicerad 
+  Registration: Registrering
```
Comment (Andycutter) [inline config/locales/se.yml]: 'registration' so everything is with small letters.
We need to change Registration to registration in the views though

**Review by Andycutter — CHANGES_REQUESTED** [review]

File: config/locales/se.yml (hunk 2 above)
Comment (ESteinman) [reply config/locales/se.yml]: Would name it;
"Kom ihåg mig" and "Registrera mig".

**Comment by tochman** [comment]
Please fix merge conflicts and address the comments.

**Pull request merged**